installation/resources/installer.yaml

```diff
@@ -178,7 +178,7 @@ spec:
           done < helm2-releases
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/kyma-installer:1.14.0-rc1
+        image: eu.gcr.io/kyma-project/develop/installer:1.14.0-rc2
```

Comment:
Why are we using the installer from `develop` directory for RC2 differently from RC1?

Reply:
resolved